administration/src/cards/extensions/__tests__/BirthdayExtension.test.tsx

```diff
@@ -15,60 +17,203 @@ const wrapper = ({ children }: { children: ReactNode }) => (
+    })
+
+    it('should show error if no birthday is provided and showRequired is true', () => {
       const { getByText } = renderWithTranslation(
         <BirthdayExtension.Component showRequired setValue={setValue} isValid={false} value={{ birthday: null }} />,
         { wrapper }
       )
+      expect(BirthdayExtension.isValid({ birthday: null })).toBeFalsy()
       expect(getByText('Bitte geben Sie ein gültiges Geburtsdatum an.')).toBeTruthy()
     })
 
-    it('should show error if provided birthday is in the future', () => {
-      const setValue = jest.fn()
-      const tomorrow = new Date()
-      tomorrow.setDate(tomorrow.getDate() + 1)
+    it('should not show error if no birthday is provided and showRequired is false', () => {
+      const { queryByText } = renderWithTranslation(
+        <BirthdayExtension.Component
+          showRequired={false}
+          setValue={setValue}
+          isValid={false}
+          value={{ birthday: null }}
+        />,
+        { wrapper }
+      )
+      expect(BirthdayExtension.isValid({ birthday: null })).toBeFalsy()
+      expect(queryByText('Bitte geben Sie ein gültiges Geburtsdatum an.')).toBeNull()
+    })
+
+    it('should show error if no birthday is provided, showRequired is false and user left field', () => {
+      const { getByText, getByPlaceholderText } = renderWithTranslation(
+        <BirthdayExtension.Component
+          showRequired={false}
+          setValue={setValue}
+          isValid={false}
+          value={{ birthday: null }}
+        />,
+        { wrapper }
+      )
+      const datePicker = getByPlaceholderText('TT.MM.JJJJ')
+      fireEvent.blur(datePicker)
+      expect(BirthdayExtension.isValid({ birthday: null })).toBeFalsy()
+      expect(getByText('Bitte geben Sie ein gültiges Geburtsdatum an.')).toBeTruthy()
+    })
+
+    it('should show error if provided birthday is too far in the past', () => {
+      const birthDayTooFarInPast = minBirthday.subtract({ days: 1 })
       const { getByText } = renderWithTranslation(
         <BirthdayExtension.Component
           showRequired
           setValue={setValue}
           isValid={false}
-          value={{ birthday: PlainDate.fromLocalDate(tomorrow) }}
+          value={{ birthday: birthDayTooFarInPast }}
+        />,
+        { wrapper }
+      )
+      expect(BirthdayExtension.isValid({ birthday: birthDayTooFarInPast })).toBeFalsy()
+      expect(getByText('Das Geburtsdatum darf nicht vor dem 01.01.1900 liegen.')).toBeTruthy()
+    })
+
+    it('should not show error if provided birthday is today', () => {
+      const { queryByText } = renderWithTranslation(
+        <BirthdayExtension.Component showRequired setValue={setValue} isValid={false} value={{ birthday: today }} />,
+        { wrapper }
+      )
+      expect(BirthdayExtension.isValid({ birthday: today })).toBeTruthy()
+      expect(queryByText('Das Geburtsdatum darf nicht in der Zukunft liegen.')).toBeNull()
```

Comment:
💭 I think it's better to check that there is no errors in general in negative tests, rather than “absence of one particular error”. 
For example, by the name or id of the component containing the errors

Reply:
okay i agree on this if different errors can be expected and you can not test against one particular error that it will not be shown.
I adjusted these cases.
But for the birthday hint, you have one particular text to check if its there or not, so this test is more explicit and i will keep it there